OpenRA.Game/FileSystem/ZipFile.cs

```diff
@@ -225,14 +221,13 @@ public static bool TryParseReadWritePackage(string filename, out IReadWritePacka
 				return false;
 			}
 
-			var s = new FileStream(filename, FileMode.Open);
-			package = new ReadWriteZipFile(s, filename);
+			package = new ReadWriteZipFile(filename);
```

Comment:
If the file didn't exist, FileNotFoundException was thrown, now it will create empty MemoryStream. Is it alright?